server/src/graphql-types/EventUser.ts

```diff
@@ -40,3 +40,18 @@ export class EventUser {
   @Field(() => User)
   user: User;
 }
+
+@ObjectType()
+export class UserEvents {
+  @Field(() => Boolean)
+  subscribed: boolean;
+
+  @Field(() => Rsvp)
+  rsvp: Rsvp;
+
+  @Field(() => EventRole)
+  event_role: EventRole;
+
+  @Field(() => Event)
+  event: Event;
+}
```

Comment:
I don't think it's necessary to create a new class. We can add `event` to `EventUser` and get it that way.

Reply:
I am worried, about the chance that the class be used somewhere it shouldn't, and we query data it shouldn't. this can be solved with tests, but lets limit the access in queries that use these class, to limit future errors, instead of relaying on memory and tests.